commit daeeceff057dd3317f1e69aebde01dd9d489d18a
Author: Natanael Copa <natanael.copa@gmail.com>
Date:   Fri Jun 12 17:11:51 2009 +0200

    Revert "ash: make dot command search current directory first, as bash does."
    
    This reverts commit 8ad78e1ec7b2e873953f9f476fb63b5893526c39.

diff --git a/shell/ash.c b/shell/ash.c
index 3452351..b62c2cd 100644
--- a/shell/ash.c
+++ b/shell/ash.c
@@ -11904,16 +11904,7 @@ find_dot_file(char *name)
 	if (strchr(name, '/'))
 		return name;
 
-	/* IIRC standards do not say whether . is to be searched.
-	 * And it is even smaller this way, making it unconditional for now:
-	 */
-	if (1) { /* ENABLE_ASH_BASH_COMPAT */
-		fullname = name;
-		goto try_cur_dir;
-	}
-
 	while ((fullname = padvance(&path, name)) != NULL) {
- try_cur_dir:
 		if ((stat(fullname, &statb) == 0) && S_ISREG(statb.st_mode)) {
 			/*
 			 * Don't bother freeing here, since it will
